### PR TITLE
Fix #86: Support multiple auth tokens

### LINKED ISSUE #86: Support multiple auth tokens
## Use cases:

* "Non downtime" rotations: add a second token temporarily, update all monitoring to use second token, remove first token
* Different tokens for different users: could enable improved access logging, reduce impact of a token leak, etc.

## Current design

Set `WATCHMAN_TOKEN` to a single token value

## Proposed design

New environment variable: `WATCHMAN_TOKENS`

This would be a comma-separated (TBD) list of tokens which are all valid.

### PR DESCRIPTION
**Pull request by mwarkentin**
* Add deprecation warning
* Update logic to handle both `WATCHMAN_TOKEN` and `WATCHMAN_TOKENS`
* Deprecate `WATCHMAN_TOKEN`
* Update readme
* Add tests for single / multiple tokens

### CODE REVIEW

File: watchman/decorators.py [hunk 1]
```diff
@@ -66,14 +66,19 @@ def _get_passed_token(request):
...
-
-        if watchman_token is None:
+        if settings.WATCHMAN_TOKENS:
+            watchman_tokens = settings.WATCHMAN_TOKENS.split(',')
+        elif settings.WATCHMAN_TOKEN:
+            watchman_tokens = [settings.WATCHMAN_TOKEN, ]
+        else:
+            watchman_tokens = []
+
+        if not watchman_tokens:
```
Comment (nduthoit) [inline watchman/decorators.py]: why not set `watchman_tokens = None` in line 74 and check `if watchman_tokens is None`?

File: watchman/decorators.py [hunk 2]
```diff
@@ -66,14 +66,19 @@ def _get_passed_token(request):
...
     def _validate_token(request):
-        watchman_token = settings.WATCHMAN_TOKEN
-
-        if watchman_token is None:
+        if settings.WATCHMAN_TOKENS:
+            watchman_tokens = settings.WATCHMAN_TOKENS.split(',')
+        elif settings.WATCHMAN_TOKEN:
+            watchman_tokens = [settings.WATCHMAN_TOKEN, ]
+        else:
+            watchman_tokens = None
```
Comment (nduthoit) [inline watchman/decorators.py]: Actually, can just `return True` here

Comment (mwarkentin) [inline watchman/decorators.py]: Good call.. thanks. 💃

**Comment by nduthoit** [comment]
LGTM 💯

**Pull request merged**